backend/helpers/emailService.js

```diff
@@ -37,7 +37,7 @@ module.exports.sendEmail = async (email, data, type) => {
       errorStack: err,
       user: email,
     });
-    return error;
+    return next(error);
```

Comment:
this should throw the error back to caller and not end the execution using next(error)